Makefile

```diff
@@ -190,3 +194,29 @@ ifeq ("$(wildcard .travis/openwhisk-utilities/scancode/scanCode.py)", "")
 endif
 	.travis/openwhisk-utilities/scancode/scanCode.py --config .travis/ASF-Release.cfg ./
 
+release-src:
```

Comment:
add some comment in this way: https://github.com/apache/apisix/pull/2481/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R188

Then we can get help information through the command `make help`.

Reply:
It is ony for Apache release manager